Header header

float64 P
float64 I
float64 D
float64 FF

float64 neg_P
float64 neg_I
float64 neg_D
float64 neg_FF

float64 min
float64 max

float64 p_component
float64 i_component
float64 d_component
float64 ff_component
float64 constant
float64 control

float64 target
float64 actual
float64 error
